templates/admin/plugins/solr.tpl

```diff
@@ -28,6 +28,10 @@
 						<div class="form-group col-sm-6">
 							<label for="port">Port</label>
 							<input class="form-control" type="text" name="port" id="port" placeholder="8983" />
+						</div>
+							<div class="form-group col-sm-6">
+							<label for="port">Secure</label>
+							<input class="form-control" type="checkbox" name="secure" id="secure">
```

Comment:
Recommend you use this style to create a checkbox that looks nicer in Bootstrap: https://github.com/julianlam/nodebb-plugin-markdown/blob/master/public/templates/admin/plugins/markdown.tpl#L10-L15

Also the label should have `for="secure"`

Reply:
> Recommend you use this style to create a checkbox that looks nicer in Bootstrap: https://github.com/julianlam/nodebb-plugin-markdown/blob/master/public/templates/admin/plugins/markdown.tpl#L10-L15
> 
> Also the label should have `for="secure"`

thx.